projects/vdk-core/src/vdk/api/plugin/plugin_input.py

```diff
@@ -237,34 +238,43 @@ def ingest_payload(
         destination_table: Optional[str],
         target: Optional[str] = None,
         collection_id: Optional[str] = None,
+        metadata: Optional[IngestionMetadata] = None,
     ) -> Optional[IngestionMetadata]:
         """
         Do the actual ingestion of the payload
 
         :param payload: List[dict]
-            The payloads to be ingested. Depending on the number of payloads to be
-            processed, there might 0 or many dict objects. Each dict object is a
-            separate payload.
+            The payloads to be ingested. Depending on the number of payloads
+            to be processed, there might be 0 or many dict objects. Each dict
+            object is a separate payload.
```

Comment:
>  Each dict  object is a separate payload.

That does not make sense ... the variable name is called payload. Payload is list of dictionary objects .. and we get recursive definition if object is a payload as well :)

Reply:
Indeed, it looks like this docstring had turned into some shady "inception"-style description. :)
Reworded to make it clearer what it actually is. Thanks